example/gluon/embedding_learning/model.py

```diff
@@ -218,6 +220,8 @@ def hybrid_forward(self, F, anchors, positives, negatives, beta_in, a_indices=No
         neg_loss = F.maximum(beta - d_an + self._margin, 0.0)
 
         pair_cnt = float(F.sum((pos_loss > 0.0) + (neg_loss > 0.0)).asscalar())
+        if pair_cnt == 0.0:
+            return F.zeros(1)
```

Comment:
I recommend this code be changed to a Block or be written to be a proper HybridBlock. There are several issues that prevent the current code to be hybridizable.